src/ray/gcs/gcs_server/stats_handler_impl.cc

```diff
@@ -34,13 +34,33 @@ void DefaultStatsHandler::HandleAddProfileData(const AddProfileDataRequest &requ
     GCS_RPC_SEND_REPLY(send_reply_callback, reply, status);
   };
 
-  Status status = gcs_client_.Stats().AsyncAddProfileData(profile_table_data, on_done);
+  Status status = gcs_table_storage_->ProfileTable().Put(UniqueID::FromRandom(),
+                                                         *profile_table_data, on_done);
   if (!status.ok()) {
     on_done(status);
   }
   RAY_LOG(DEBUG) << "Finished adding profile data, component type = "
                  << request.profile_data().component_type() << ", node id = " << node_id;
 }
 
+void DefaultStatsHandler::HandleGetAllProfileInfo(
+    const rpc::GetAllProfileInfoRequest &request, rpc::GetAllProfileInfoReply *reply,
+    rpc::SendReplyCallback send_reply_callback) {
+  RAY_LOG(INFO) << "Getting all profile info.";
```

Comment:
Can you change this to DEBUG? INFO is gonna be very verbose.

Reply:
OK